streampipes-client-python/mkdocs.yml

```diff
@@ -79,6 +79,10 @@ plugins:
       version_selector: true
       css_dir: css
       javascript_dir: js
+  - search:
+      lang:
+        - en
+      seperator: '[\s\-,:_!=\[\]()"/]+'
```

Comment:
Why did you only special chars as separator?
We are on version 8 of mkdocs-material so the default seperator is `'[\s\-\.]+'`
What's your idea about changing this? :)

Reply:
I thought user might not know the exact thing to search, may end up adding some special character. If you want to i can remove them and keep the defaults.

Reply:
Sounds valid
Let's try it like this